media-sound/vimpc/vimpc-0.09.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools
+
+DESCRIPTION="An ncurses based mpd client with vi-like key bindings"
+HOMEPAGE="https://github.com/boysetsfrog/vimpc"
+SRC_URI="https://github.com/boysetsfrog/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="boost taglib"
+
+RDEPEND="dev-libs/libpcre
+	media-libs/libmpdclient
+	boost? ( dev-libs/boost )
+	taglib? ( media-libs/taglib )"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+DOCS=( AUTHORS README.md doc/vimpcrc.example )
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
really necessary for the release version?

Reply:
Unfortunately, yes. Otherwise it fails with
```
 * ERROR: media-sound/vimpc-0.09.2::gentoo failed (configure phase):
 *   no configure script found
```